src/main/java/com/boundary/plugin/sdk/EventSinkAPI.java

```diff
@@ -11,17 +11,25 @@
 // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 // See the License for the specific language governing permissions and
 // limitations under the License.
-
 package com.boundary.plugin.sdk;
 
+import com.boundary.plugin.sdk.rpc.RPC;
+
 public class EventSinkAPI implements EventSink {
-   
-    public EventSinkAPI() {
 
+    private final EventFormatter formatter;
+
+    public EventSinkAPI() {
+        formatter = new EventFormatter();
     }
 
+    @Override
     public void emit(Event event) {
-        // TODO: Implement this.
+        RPC.send(formatter.format(event));
```

Comment:
Rather, why dont you call `emit(formatter.format(event));`

With this, whenever you change the mechanism to make RPC call, you can do it in only one place